x-pack/plugins/fleet/common/services/validate_package_policy.ts

```diff
@@ -266,6 +266,18 @@ export const validatePackagePolicyConfig = (
     }
   }
 
+  if (
```

Comment:
I am wondering if we should have some unit tests for validating this, I guess we want to validate, number, yaml the same way

Reply:
We do have some tests or this file in https://github.com/elastic/kibana/blob/cd7f26dd81f6b2899e911fb6e24ab95f02f886f9/x-pack/plugins/fleet/common/services/validate_package_policy.test.ts

I updated these tests to ensure we hit the "non-boolean value" validation added in this PR.